scss/_patterns_strip.scss

```
@@ -104,7 +106,7 @@
 }
 
 @mixin vf-p-strip-shallow {
-  [class*='p-strip'].is-shallow {
+  [class*='p-strip']:not([class*='suru']).is-shallow {
```

To avoid situation that `suru` is part of some longer word I'd add a dash before it `-suru`.

```suggestion
  [class*='p-strip']:not([class*='-suru']).is-shallow {
```

Also it would be nice to add a code comment saying why we are adding this exception. Something like: `// Suru strips were not designed to be shallow, so they are excluded`